chrome/background.js

```diff
@@ -21,6 +21,17 @@ function fillLoginForm(login) {
       });
     }
   );
+
+  chrome.tabs.executeScript(
```

Comment:
could you wrap this in an `if` so that this injection happens if and only if the `login` object actually contains an OTP token?